docs/integrations/flutter-specific-workflow-steps/flutter-test.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Flutter Test
+metaTitle: Flutter Test
+metaDescription: Flutter Test
+sidebar_position: 4
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Flutter Test
+
+This component allows you to run [**Flutter Unit Tests**](https://docs.flutter.dev/cookbook/testing/unit/introduction#run-tests-in-a-terminal). Please note that it requires the preceding [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) and [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) steps to run.
+
+
+### Prerequisites
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) | This step will clone your project through the connected git provider and create the `$AC_REPOSITORY_DIR` variable. |
+| [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) | This step will install the Flutter SDK. If a version is not specified, it will install the latest **stable** version. |
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2853-testOrder1.png' />
+
+:::warning
+This step is particularly dependent on the Flutter Install step. If the Flutter SDK is not installed, the step will give an error that the required command was not found.
+:::
+
+### Input Variables
+
+You can find all the parameters required for this step in the table below, with their descriptions in detail.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2853-testInput.png' />
+
+| Variable Name                 	       | Description                         | Status 			|
+|-------------------------------|------------------------------------------------|------------------|
+| `$AC_FLUTTER_PROJECT_DIR`     | This parameter is used as the repository path. This path is created immediately after the Git Clone step. If the Git Clone step is not used, this path cannot be found. | Required|
+| `$AC_FLUTTER_JUNIT_REPORTS`       | If this is set to `YES`, [JUnit](https://junit.org/junit5/) Report will be created at AC_TEST_RESULT_PATH | Optional |
+| `$AC_FLUTTER_TEST_EXTRA_ARGS`     | You can use this parameter if you want to add an extra parameter to the build command line. | Optional |
+
+### Output Variables
+
+| Variable Name                 	       | Description                         |
+|-------------------------------|------------------------------------------------|
+| `$AC_TEST_RESULT_PATH`        | This parameter is used as the repository path. This path is created immediately after the Git Clone step. If the Git Clone step is not used, this path cannot be found. | 
```

Comment:
It seems there is no dependencies between `Git Clone` can you check this info again?
line is [here](https://github.com/appcircleio/appcircle-flutter-test-component/blob/48616a12063dde099dbb60f9300f88026e3fb52f/main.rb#L38C1-L41C6):
```
  reports_folder = File.join(ENV['AC_OUTPUT_DIR'], 'flutter_reports')
  File.open(ENV['AC_ENV_FILE_PATH'], 'a') do |f|
    f.puts "AC_TEST_RESULT_PATH=#{reports_folder}"
  end
```